net-im/signal-cli-bin/signal-cli-bin-0.13.2-r1.ebuild

```diff
@@ -18,7 +18,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 
 DEPEND="
-	|| ( virtual/jdk:21 virtual/jre:21 )
+	virtual/jre:21
 "
 RDEPEND="${DEPEND}"
```

Comment:
Same as above.
And the whole [addition here,](https://github.com/madmartin/gentoo/commit/8f8509cbc63833d9b7a302567a639fd6b2a05b9b#diff-139f95dca9ac1a0b997ed6d3b522517cc2803877c91a826db42f9dab296ec60fR20-R26), is it really needed? Never seen such in any other Java -bin package.

Reply:
It is needed, believe me.